src/development/docker-compose.yml

```diff
@@ -35,6 +35,18 @@ services:
       context: ./../Altinn.Platform/Altinn.Platform.PDF/
     extra_hosts:
       - "host.docker.internal:host-gateway"
+  altinn_pdf_service:
+    container_name: altinn-pdf-service
+    image: browserless/chrome:1-puppeteer-19.2.2
+    restart: always
+    profiles:
+        - pdf
```

Comment:
Won't adding a profile preventing pdf from starting by default when doing docker `compose up`?

Reply:
Yes, that is why I suggested it to be added in https://github.com/Altinn/altinn-studio/pull/9459#discussion_r1055523909

My intention was for this to be in place for testing, but removed when we expect everyone to use the new pdf conatiner.

Reply:
By "hiding" the new pdf service behind a profile until we have a version of the nugets that supports the new service and it's used by a nuber of apps. We should not default to consume more resources on the developers system than we need. When both the old and new are used we can enable both services as default as there is no problem that they both are running at the same time.